File: Easy/3541. Find Most Frequent Vowel and Consonant/solution.cs
public class Solution {
    public int MaxFreqSum(string s) {
        int[] characters = new int[26];
        int max_vowel = 0;
        int max_consonant = 0;

        for(int i = 0; i < s.Length; i++){
            characters[s[i] - 'a'] += 1;
            if("aiueo".Contains(s[i])){
                max_vowel = Math.Max(max_vowel, characters[s[i] - 'a']);
            } else {
                max_consonant = Math.Max(max_consonant, characters[s[i] - 'a']);
            }
        }

        return max_consonant + max_vowel;
    }
}

File: Easy/3541. Find Most Frequent Vowel and Consonant/solution.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        d = defaultdict(int)
        max_vowel = 0
        max_consonant = 0

        for i in s:
            d[i] += 1
            if i in 'aiueo':
                if d[i] > max_vowel:
                    max_vowel = d[i]
            else:
                if d[i] > max_consonant:
                    max_consonant = d[i]

        return max_vowel + max_consonant
